content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
this index file is handled in #20261 
so no need to create new file to reduce conflict